public/scripts/mmfilter.js

```diff
@@ -0,0 +1,93 @@
+(function(module){
+
+  var Filters = function(instance){
```

Comment:
you could ditch this whole constructor by using a .map to get the properties you need from your existing Cars objects